opentech/apply/funds/views.py

```diff
@@ -222,3 +227,104 @@ def form_valid(self, form):
         self.object.form_data = form.cleaned_data
         self.object.save()
         return HttpResponseRedirect(self.get_success_url())
+
```

Comment:
Hmm.. this views.py is getting awfully big. Split into component views in a views dir?

Reply:
Agreed, considering separating application submission to new app.